pkg/csi_driver/controller.go

```diff
@@ -214,7 +214,14 @@ func (s *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolu
 		}
 		if createErr != nil {
 			glog.Errorf("Create volume for volume Id %s failed: %v", volumeID, createErr)
-			return nil, status.Error(codes.Internal, createErr.Error())
+			// Return InvalidArgument code for googleapi errors that are caused by users such as Error 429: Quota limit exceeded
```

Comment:
nit: move this comment to the IsUserError() ftn definition (eg, if we called this ftn a second time from somewhere else we wouldn't want to copy the comment)